### PR TITLE
[KOGITO-8908] Systout custom type using log framework

### PR DESCRIPTION
**Pull request by fjtirado**
Rather than doing System.out.println.
Doc PR: https://github.com/kiegroup/kogito-docs/pull/321

### CODE REVIEW

File: kogito-serverless-workflow/kogito-serverless-workflow-runtime/src/main/java/org/kie/kogito/serverless/workflow/actions/SysoutAction.java [hunk 1]
```diff
@@ -16,14 +16,19 @@
...
+import org.kie.kogito.internal.utils.ConversionUtils;
 
 public class SysoutAction extends BaseExpressionAction {
-    public SysoutAction(String lang, String expr, String inputVar) {
+
+    private final WorkflowLogLevel logLevel;
+
+    public SysoutAction(String lang, String expr, String inputVar, String level) {
         super(lang, expr, inputVar);
+        logLevel = ConversionUtils.isEmpty(level) ? WorkflowLogLevel.INFO : WorkflowLogLevel.valueOf(level);
```
Comment (hbelmiro) [inline kogito-serverless-workflow/kogito-serverless-workflow-runtime/src/main/java/org/kie/kogito/serverless/workflow/actions/SysoutAction.java:27]: Shouldn't we use the same default log level as Quarkus'?

File: kogito-serverless-workflow/kogito-serverless-workflow-runtime/src/main/java/org/kie/kogito/serverless/workflow/actions/WorkflowLogger.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ * limitations under the License.
+ */
+package org.kie.kogito.serverless.workflow.actions;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class WorkflowLogger {
+
+    private static final Logger logger = LoggerFactory.getLogger(SysoutAction.class);
```
Comment (hbelmiro) [inline kogito-serverless-workflow/kogito-serverless-workflow-runtime/src/main/java/org/kie/kogito/serverless/workflow/actions/WorkflowLogger.java:23]:
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(WorkflowLogger.class);
```

File: kogito-serverless-workflow/kogito-serverless-workflow-runtime/src/main/java/org/kie/kogito/serverless/workflow/actions/SysoutAction.java (hunk 1 above)
Comment (fjtirado) [reply kogito-serverless-workflow/kogito-serverless-workflow-runtime/src/main/java/org/kie/kogito/serverless/workflow/actions/SysoutAction.java:27]: This is quarkus independent code

File: kogito-serverless-workflow/kogito-serverless-workflow-runtime/src/main/java/org/kie/kogito/serverless/workflow/actions/WorkflowLogger.java (hunk 2 above)
Comment (fjtirado) [reply kogito-serverless-workflow/kogito-serverless-workflow-runtime/src/main/java/org/kie/kogito/serverless/workflow/actions/WorkflowLogger.java]: @hbelmiro That was right, but I changed it a bit after adding more test, please take another look

**Review by hbelmiro — APPROVED** [review]

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**